app/views/reports/_review_report.html.erb

```diff
@@ -58,7 +58,7 @@
     <tbody>
 
     <!-- Set the review data and row span for each reviewer -->
-    <% sort_reviewer_by_review_volume_desc.each do |reviewer| %>
+    <% sort_reviewer_by_review_volume_desc.each_with_index do |reviewer, index| %>
       <% @response_maps, @rspan = get_data_for_review_report(@id, reviewer.id, @type) %>
       <tr>
         <% if reviewer.id != -1 %>
```

Comment:
this can also be converted into something with an unless
try `unless reviewer.id == 1`